src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -165,7 +164,7 @@ export const GithubConfigurationGet = async (req: Request, res: Response, next:
 			return;
 		}
 
-		const gitHubAppClient = await createAppClient(log, jiraHost);
+		const gitHubAppClient = await createAppClient(req.body.gitHubInstallationId, log, jiraHost);
```

Comment:
1. The function is called `GithubConfigurationGet`, so I assume this route is a `HTTP GET` request. And in case of those request, I don’t think using `req.body` is a good way. Because `GET` request usually is NOT supposed to have a body? Possibly solution is to attach the id to url, like `/github/configiguration?ghid=xxxx`?

2. Second, will it break the existing github cloud? As I don’t see any gitHubInstallationId in the existing github-configuration page (see screenshot)

<img width="1559" alt="image" src="https://user-images.githubusercontent.com/105693507/173492263-e375126a-91ca-453e-a62b-cf56978c4493.png">